shared/src/main/java/uk/gov/di/authentication/shared/helpers/SessionIdHelper.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.di.authentication.shared.helpers;
+
+import java.util.Map;
+import java.util.Objects;
+
+public class SessionIdHelper {
```

Comment:
We should add some tests for this new helper class

Reply:
Added in the last commit SessionIdHelperTest.java let me know if we should add any more cases

Reply:
Great! Let's squash the test in to the commit where the behaviour is introduced, it's good to have implementation and relevant tests in the same commit